src/main/java/spotify/playback/data/lyrics/GeniusLyricsScraper.java

```diff
@@ -47,37 +58,96 @@ public String getSongLyrics(String artistName, String songName) {
   }
 
   private String findLyricsUrl(String artistName, String songName) throws IOException {
-    String searchUrl = "https://genius.com/api/search?q=" + artistName + " " + songName;
+    // Preprocess artistName and songName to remove brackets
+    String processedArtistName = preprocessString(artistName);
+    String processedSongName = preprocessString(songName);
+
+    String searchUrl = "https://genius.com/api/search?q=" + processedArtistName + " " + processedSongName;
+    System.out.println(searchUrl);
+
     Connection.Response response = Jsoup.connect(searchUrl)
-      .userAgent(USER_AGENT)
-      .ignoreContentType(true)
-      .execute();
+            .userAgent(USER_AGENT)
+            .ignoreContentType(true)
+            .execute();
     String json = response.body();
 
-    String matchingPath = findMatchingPath(json, artistName, songName);
+    String matchingPath = findMatchingPath(json, processedArtistName, processedSongName);
+    System.out.println(matchingPath);
+
     if (matchingPath != null) {
       return "https://genius.com" + matchingPath;
     }
     return null;
   }
 
+  private String preprocessString(String input) {
+    // Remove all occurrences of problematic symbols
+    input = input.replace("#", "");
```

Comment:
These three lines can be shorted into one using replaceAll and regex:

    input = input.replaceAll("[#@_]", "");